selfie.c

```diff
@@ -4136,5 +4301,6 @@ int main(int argc, int *argv) {
         }
     } else
         // default: compiler
-        main_compiler();
-}
+        //main_compiler();
+	testList();
```

Comment:
Do not alter the default behaviour of selfie. Add, for example, a new flag to trigger your test case.